handlers/handler.go

```diff
@@ -88,6 +91,14 @@ func HandleRequest(filterRequest event.FilterRequest) FilterResponse {
 		panic(err)
 	}
 
+	defer func() {
+		if r := recover(); r != nil {
+			fmt.Printf("Something went wrong: %s\n%s", r, debug.Stack())
```

Comment:
Won't `debug.Stack()` here always be the same stack trace (of this deferred recover block), rather than the stack trace of where the error occurred originally?